kebechet/managers/manager.py

```diff
@@ -31,6 +31,8 @@
 from ogr.services.base import BaseGitService
 from ogr.abstract import Issue, PullRequest
 
+from kebechet import utils
```

Comment:
this is not stated in __init__.py , how do this work ?
should this be , from kebechet.utils import get_issue_by_title .

Reply:
This works fine, utils should be discoverable as a module by file name alone. I left it as `import utils` so the distinction between `self.get_issue_by_title` and `utils.get_issue_by_title` was more clear.